packages/cli/src/runAndroid/runAndroid.js

```diff
@@ -47,14 +48,14 @@ function runAndroid(argv: Array<string>, config: ContextT, args: Object) {
 
   return isPackagerRunning(args.port).then(result => {
     if (result === 'running') {
-      console.log(chalk.bold('JS server already running.'));
+      logger.info(chalk.bold('JS server already running.'));
     } else if (result === 'unrecognized') {
-      console.warn(
+      logger.warn(
         chalk.yellow('JS server not recognized, continuing with build...')
```

Comment:
```suggestion
       'JS server not recognized, continuing with build...'
```

Reply:
this (and similar) still need to be addressed

Reply:
fixed